### PR TITLE
[MIG] Migration to Odoo V13.0

### PR DESCRIPTION
**Pull request by ougc27**
Hello @alan196 

This PR contains some snippet updates to make the extension compatible with Odoo V13.0
Could you check it please?

Thank you!

### CODE REVIEW

File: Common/xml/Base/base-xml-odoo.sublime-snippet [hunk 1]
```diff
@@ -3,9 +3,7 @@
 <snippet>
     <content><![CDATA[<?xml version="1.0" encoding="UTF-8"?>
 <odoo>
-    <data>
-        $0
-    </data>
+    $0
 </odoo>
```
Comment (alan196) [inline Common/xml/Base/base-xml-odoo.sublime-snippet:7]: Could you please add a new line to be added at the end of the file 🙏

File: Common/python/Methods/method-copy-odoo.sublime-snippet [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!-- Odoo Python Snippets -->
 
 <snippet>
-    <content><![CDATA[@api.multi
+    <content><![CDATA[
     def copy(self, default=None):
```
Comment (alan196) [inline Common/python/Methods/method-copy-odoo.sublime-snippet]: start the `def ` just after the `CDATA[def copy(self, default=None):`

If you don't do this every time you use this snippet you will have an extra identation

File: Common/xml/Objects/object-form-view.sublime-snippet [hunk 3]
```diff
@@ -1,25 +1,25 @@
 <snippet>
-	<content><![CDATA[
+    <content><![CDATA[
 <record model="ir.ui.view" id="${1:name_id}_form">
```
Comment (alan196) [inline Common/xml/Objects/object-form-view.sublime-snippet]: the same af the def in python methods... is better to have this just after de CDATA[

**Review by alan196 — CHANGES_REQUESTED** [review]
@ougc27 Thanks for your contribution 👍 I make some comments

**Comment by ougc27** [comment]
Hello @alan196 the requested changes are done :+1:

**Comment by alan196** [comment]
Thanks I'll merge... I also will make another fast change I will notify you when it's done

**Pull request merged**